outputs.tf

```diff
@@ -1,4 +1,16 @@
+data "aws_caller_identity" "current" {}
+data "aws_region" "current" {}
+
+locals {
+  account_id = data.aws_caller_identity.current.account_id
+  region     = data.aws_region.current.name
+}
+
 output "recipe_arn" {
   description = "ARN of the EC2 Image Builder Recipe"
-  value       = aws_cloudformation_stack.this.outputs["RecipeArn"]
+  value       = "arn:aws:imagebuilder:${local.region}:${local.account_id}:image-recipe/${lower(var.name)}/${var.recipe_version}"
```

Comment:
Does this work even with the name-uuid()?

Reply:
Yes. That only adjusts the name of the CloudFormation stack, not the name of the underlying recipe. The recipe is still getting the raw name.